app/validators/date_relative_validator.rb

```diff
@@ -0,0 +1,27 @@
+class DateRelativeValidator < ActiveModel::EachValidator
+  def initialize(in_the_past: nil, **kwargs)
+    @in_the_past = in_the_past
+    super(**kwargs)
+  end
+
+  def validate_each(record, attribute, value)
+    if future_date?(value)
+      record.errors.add(attribute, :less_than)
+    end
+  end
+
+  private
+
+  attr_reader :in_the_past
+
+  def future_date?(value)
+    return false if in_the_past.nil?
```

Comment:
We really need to start running rubocop again - it will highlight things like this.

All guard clauses should have an empty line before the normal code begins - just makes for easier reading